c7n/resources/account.py

```diff
@@ -434,3 +434,59 @@ def process(self, resources, event=None):
             resources[0]['c7n:ServiceLimitsExceeded'] = exceeded
             return resources
         return []
+
+
+
+@actions.register('request-limit-increase')
+class RequestLimitIncrease(BaseAction):
+    """ File support ticket to raise limit
+
+    :Example:
+
+    .. code-block: yaml
+
+        policies:
+          - name: account-service-limits
+            resource: account
+            filters:
+             - type: service-limit
+               services:
+                 - EBS
+               threshold: 60.5
+             actions:
+               - type: request-limit-increase
+                 notify: [email, email2]
+                 percent-increase: 50
+                 template: foo | sane default
+    """
+
+    schema = type_schema(
+        'request-limit-increase',
+        **{'notify': {'type': 'array'},
+           'percent-increase': {'type': 'number'},
+           'template': {'type': 'string'}})
+
+    default_template = 'Please raise the limit of {id} by {percent}%'
+    
+    def process(self, resources):
+        session = local_session(self.manager.session_factory)
+        client = session.client('account')
+        
+        for resource in resources:
+            resource_id = resource.get('VolumeId'),
+
+            subject = 'Raise the limit of {}'.format(resource_id)
```

Comment:
this seems a bit odd, not sure why volumeid would be present, its an account limit we want to increase, not that of the individual volume which is self service. ie let's say i default to 50tb data limit on the account, and i do that spread across a 100 500gb volumes, i want to increase the total volume of the account, not that of an individual volume. the resource here is the actual account itself.